files/usr/share/cinnamon/cinnamon-settings/cinnamon-settings.py

```diff
@@ -91,7 +91,8 @@
     [_("Package Management"),            "yumex-dnf",                           "cs-sources",         "admin",          _("update, install, repository, package, source, download")],
     [_("Users and Groups"),              "cinnamon-settings-users",             "cs-user-accounts",   "admin",          _("user, users, account, accounts, group, groups, password")],
     [_("Bluetooth"),                     "blueberry",                           "cs-bluetooth",       "hardware",       _("bluetooth, dongle, transfer, mobile")],
-    [_("Manage Services and Units"),     "systemd-manager-pkexec",              "cs-sources",         "admin",          _("systemd, units, services, systemctl, init")]
+    [_("Manage Services and Units"),     "systemd-manager-pkexec",              "cs-sources",         "admin",          _("systemd, units, services, systemctl, init")],
+    [_("Disks Management"),              "gnome-disks",                         "disks",              "hardware",          _("disk, disk, manage, hardware, management")]
```

Comment:
Hello, everybody.

@pyroxar: Here you added the **disk** keyword twice (maybe you wanted to add its plural (**disks**)?). Also, a couple of more _relevant_ keywords wouldn't _hurt_ (e.g. **hdd**, **hard drive**, **storage**).

Reply:
I fixed it.